### ISSUE TITLE
autoreconf failed on macOS Mojave

### ISSUE THREAD

**Issue by qzivli** [wingo/fibers#24]
```
autoreconf -vif
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force -I m4
autoreconf: configure.ac: tracing
autoreconf: running: glibtoolize --copy --force
glibtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, 'build-aux'.
glibtoolize: copying file 'build-aux/ltmain.sh'
glibtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
glibtoolize: copying file 'm4/libtool.m4'
glibtoolize: copying file 'm4/ltoptions.m4'
glibtoolize: copying file 'm4/ltsugar.m4'
glibtoolize: copying file 'm4/ltversion.m4'
glibtoolize: copying file 'm4/lt~obsolete.m4'
autoreconf: running: /usr/local/Cellar/autoconf/2.69/bin/autoconf --force
configure:13082: error: possibly undefined macro: AC_LIB_LINKFLAGS_FROM_LIBS
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
autoreconf: /usr/local/Cellar/autoconf/2.69/bin/autoconf failed with exit status: 1
```

OS version: macOS Mojave (10.14.1)
Guile version: 2.2.4 (installed by homebrew)

**Comment by call-cc** [comment]
Got the same error.

MacOS version: 10.14.3
Guile version: 2.2.4 (via homebrew)

**Comment by aconchillo** [comment]
Currently, OS X is not supported. There's initial work for FreeBSD that will also be useful for OS X (since it also uses `kqueue`), see https://github.com/wingo/fibers/pull/27.

**Comment by aconchillo** [comment]
Also, running `./autogen.sh` (i.e. `autoreconf`) works fine in `master` in Catalina, OS X 10.15. You wouldn't get much further per the comment above, but at least this error doesn't show up for me. Would it be possible for you to retry?

**Comment by aconchillo** [comment]
This will be fixed when #42 gets merged.

**Comment by emixa-d** [comment]
As the new version of #42 is merged, I'm assuming this can be closed (if not, just add an appropriate comment; GitHub restricts reopening).
